indexing-service/src/test/java/org/apache/druid/indexing/input/DruidSegmentReaderTest.java

```diff
@@ -683,6 +684,117 @@ public void close()
     Assert.assertTrue("Sequence is not closed", isSequenceClosed.booleanValue());
   }
 
+  @Test
+  public void testArrayColumns() throws IOException
+  {
+    // Write a segment with two rows in it, with columns: s (string), d (double), cnt (long), met_s (complex).
```

Comment:
This comment doesn't really align with the code that I see below it.  The names of things seem stale?  Is it copy pasta from somewhere?

Reply:
heh oops yes copied some stuff from another test to get the basic outline and then added arrays